pkg/api/short_urls.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"strings"
+
+	"github.com/grafana/grafana/pkg/api/dtos"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/shorturls"
+)
+
+// POST /api/goto
+func (hs *HTTPServer) CreateShortURL(c *models.ReqContext, cmd dtos.CreateShortURLForm) Response {
+	service := shorturls.NewShortURLService(c.SignedInUser)
+	result, err := service.CreateShortURL(strings.TrimPrefix(cmd.Path, "/"))
+
+	if err != nil {
```

Comment:
Please co-locate error checks with their origin:
```suggestion
	result, err := service.CreateShortURL(strings.TrimPrefix(cmd.Path, "/"))
	if err != nil {
```